libs/orders/src/lib/components/order-list-manager/use-order-list-data.ts

```diff
@@ -77,11 +77,9 @@ export const useOrderListData = ({
     update,
     insert,
     variables,
+    updateOnInit: true,
   });
-  if (!dataRef.current && data) {
-    totalCountRef.current = totalCount;
-    dataRef.current = data;
-  }
+  totalCountRef.current = totalCount;
```

Comment:
chore: This could be also moved to update function, just for consistency

Reply:
yeah, but it's rather outside scope of this issue..